### ISSUE TITLE
Can't get mono installation working on Debian 9

### ISSUE THREAD

**Issue by Fivefor** [mono/mono#8590]
Hi,

I'm trying to install Mono on my Debian Linux according to this guide:
https://www.mono-project.com/download/stable/#download-lin-debian

But when I try to execute first line:
sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF

it gives me this:
Executing: /tmp/apt-key-gpghome.wFSfsOnNzG/gpg.1.sh --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
gpg: failed to start the dirmngr '/usr/bin/dirmngr': File or folder does not exist
gpg: connecting dirmngr at '/tmp/apt-key-gpghome.wFSfsOnNzG/S.dirmngr' failed: File or folder does not exist
gpg: keyserver receive failed: No dirmngr

**Comment by directhex** [comment]
Yeah, looks like the default packages in Debian 9 chroots have changed since I wrote the instructions. Will fix up docs.

**Comment by directhex** [comment]
Docs updated.

**Comment by Fivefor** [comment]
Thanks!
